src/coreimage.cs

```diff
@@ -5673,85 +5498,3365 @@ interface CIKeystoneCorrection {
+
+		[Abstract]
+		[NullAllowed, Export ("inputImage", ArgumentSemantic.Retain)]
+		[CoreImageFilterProperty ("inputImage")]
+		CIImage InputImage { get; set; }
+	}
+
+	interface ICISepiaToneProtocol {}
+
+	[iOS (13,0)]
+	[TV (13,0)]
+	[Mac (10,15)]
+	[Protocol (Name = "CISepiaTone")]
+	interface CISepiaToneProtocol : CIFilterProtocol {
+
+		[Abstract]
+		[NullAllowed, Export ("inputImage", ArgumentSemantic.Retain)]
+		CIImage InputImage { get; set; }
+
+		[Abstract]
+		[Export ("intensity")]
+		float Intensity { get; set; }
+	}
+
+	interface ICIShadedMaterialProtocol {}
+
+	[iOS (13,0)]
+	[TV (13,0)]
+	[Mac (10,15)]
+	[Protocol (Name = "CIShadedMaterial")]
+	interface CIShadedMaterialProtocol : CIFilterProtocol {
+
+		[Abstract]
+		[NullAllowed, Export ("inputImage", ArgumentSemantic.Retain)]
+		CIImage InputImage { get; set; }
+
+		[Abstract]
+		[NullAllowed, Export ("shadingImage", ArgumentSemantic.Retain)]
+		CIImage ShadingImage { get; set; }
+
+		[Abstract]
+		[Export ("scale")]
+		float Scale { get; set; }
+	}
+
+	interface ICISharpenLuminanceProtocol {}
+
+	[iOS (13,0)]
+	[TV (13,0)]
+	[Mac (10,15)]
+	[Protocol (Name = "CISharpenLuminance")]
+	interface CISharpenLuminanceProtocol : CIFilterProtocol {
+
+		[Abstract]
+		[NullAllowed, Export ("inputImage", ArgumentSemantic.Retain)]
+		CIImage InputImage { get; set; }
+
+		[Abstract]
+		[Export ("sharpness")]
+		float Sharpness { get; set; }
+
+		[Abstract]
+		[Export ("radius")]
+		float Radius { get; set; }
+	}
+
+	interface ICISixfoldReflectedTileProtocol {}
+
+	[iOS (13,0)]
+	[TV (13,0)]
+	[Mac (10,15)]
+	[Protocol (Name = "CISixfoldReflectedTile")]
+	interface CISixfoldReflectedTileProtocol : CIFilterProtocol {
+
+		[Abstract]
+		[NullAllowed, Export ("inputImage", ArgumentSemantic.Retain)]
+		CIImage InputImage { get; set; }
+
+		[Abstract]
+		[Export ("center", ArgumentSemantic.Assign)]
+		CGPoint InputCenter { get; set; }
+
+		[Abstract]
+		[Export ("angle")]
+		float Angle { get; set; }
+
+		[Abstract]
+		[Export ("width")]
+		float Width { get; set; }
+	}
+
+	interface ICISixfoldRotatedTileProtocol {}
+
+	[iOS (13,0)]
+	[TV (13,0)]
+	[Mac (10,15)]
+	[Protocol (Name = "CISixfoldRotatedTile")]
+	interface CISixfoldRotatedTileProtocol : CIFilterProtocol {
+
+		[Abstract]
+		[NullAllowed, Export ("inputImage", ArgumentSemantic.Retain)]
+		CIImage InputImage { get; set; }
+
+		[Abstract]
+		[Export ("center", ArgumentSemantic.Assign)]
+		CGPoint InputCenter { get; set; }
+
+		[Abstract]
+		[Export ("angle")]
+		float Angle { get; set; }
+
+		[Abstract]
+		[Export ("width")]
+		float Width { get; set; }
+	}
+
+	interface ICISmoothLinearGradientProtocol {}
+
+	[iOS (13,0)]
+	[TV (13,0)]
+	[Mac (10,15)]
+	[Protocol (Name = "CISmoothLinearGradient")]
+	// `CILinearGradientProtocol` is a bit of a lie - but it would not compile (registrar) otherwise
+	interface CISmoothLinearGradientProtocol : CILinearGradientProtocol {
+
+		/* we get those from ICILinearGradientProtocol
```

Comment:
So if we're keeping this code in case it's ever added again, wouldn't a `#if false` be better?
Similar to my comment above I don't know if we want dead code (:

Reply:
Good catch, left over from the original attempt. I think that's fixed (properly) now. I'll uncomment it

Reply:
uncommented (that scenario works fine now)